frontend/website/pages/docs/archive-node.mdx

```diff
@@ -0,0 +1,129 @@
+import Page from '@reason/pages/Docs';
+export default Page({title: "Archive Node"});
+
+# Archive Node
+Coda nodes are succinct by default, meaning they don't need to maintain historical information about the network, block, or transactions. 
+For some usecases, it is useful to maintain this historical data, which you can do by running an **Archive Node.**
+
+An **Archive Node** is just a regular Coda daemon that is connected to a running archive process. 
+The daemon will regularly send blockchain data to the archive process, 
+which will store it in a [Postgres](https://www.postgresql.org/) database. 
+
+Running an archive node therefore requires some knowledge of managing a Postgres database instance.
+In this section, we'll set up a database, run the Archive Node, connect it to a daemon, and try some queries on the data. 
+Let's get started by installing what we need.  
+
+## Installation 
+1. Install the latest version of Coda. If you haven't upgraded to the latest version of the daemon, head [back to the docs](/docs/getting-started) to get the latest version. You can run ```coda -help``` to check if the installation succeeded.
+2. Install [Postgres](https://www.postgresql.org/download/).
+    - MacOS: Install using [Homebrew](https://wiki.postgresql.org/wiki/Homebrew). ```brew install postgresql```. That will give you both the Postgres server and the command line interface, **psql**. 
+3. Install the archive node package. 
+    - MacOS: Install using [Homebrew](https://brew.sh).
+    ```
+    brew install codaprotocol/coda/coda-archive
+    ```
+    - Windows: (placeholder)
+    - Ubuntu/Debian: (placeholder)
```

Comment:
```suggestion
    - Ubuntu/Debian:
      ```
      sudo apt-get install -t unstable coda-archive
      ```
    - Docker: Use the following container
      ```
      codaprotocol/coda-daemon:0.0.16-beta5-master-fdc283b and coda-archive:0.0.16-beta5-master-fdc283b
      ```
```